scenarios/centos-9/ceph_backends.yml

```diff
@@ -12,10 +12,10 @@ cifmw_make_ceph_environment:
   CEPH_DATASIZE: "10Gi"
 
 pre_deploy:
-  - name: Ceph deploy
+  - name: 81 Ceph deploy
     type: playbook
     source: ceph-deploy.yml
-  - name: Kustomize OpenStack CR with Ceph
+  - name: 82 Kustomize OpenStack CR with Ceph
```

Comment:
TODO: add source: control_plane_hci_pre_deploy.yml here too, used in ceph-toy scenario only.